data/DMHiggsAnalysis.cfg

```diff
@@ -13,13 +13,12 @@ SampleName:       sample
 InputFileList:    input_data.txt
 BaseConfig:       HGamAnalysisFramework/HGamRel20p7.config
 Include:          HGamAnalysisFramework/MCSamples.config
-
+NTupleName:	  ggH125_gamgam.NTuple.root
```

Comment:
okay, then I understand the situation, just to double check, if you run on MxAOD, get the genMET from  all non-interacting particles. And the same signal sample, but DxAODs, get the genMET from DM particles, are the two plots same ?